src/main/java/com/jfrog/ide/common/persistency/ScanCacheObject.java

```diff
@@ -0,0 +1,34 @@
+package com.jfrog.ide.common.persistency;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.jfrog.ide.common.nodes.FileTreeNode;
+import lombok.Getter;
+
+import java.util.List;
+
+/**
+ * The implementation of the scan cache. Contains a version and a map.
+ * In case of incorrect version, it zaps the cache and starts over a new map.
+ */
+@Getter
+public class ScanCacheObject {
+    static int CACHE_VERSION = 4;
+
+    @JsonProperty("version")
+    int version = CACHE_VERSION;
+
+    @JsonProperty("fileTreeNodes")
+    List<FileTreeNode> fileTreeNodes;
+    @JsonProperty("scanTime")
+    long scanTime;
```

Comment:
Isn't it a scan timestamp?
```suggestion
    @JsonProperty("scanTimestamp")
    long scanTimestamp;
```